packages/manager/modules/new-billing/src/autoRenew/autorenew.service.js

```diff
@@ -202,6 +204,16 @@ export default class {
     }).$promise;
   }
 
+  putServiceV6(service, data = {}) {
+    return this.OvhHttp.put('/services/{id}', {
+      rootPath: 'apiv6',
+      urlParams: {
+        id: service.id,
+      },
+      data,
+    });
```

Comment:
```suggestion
    return this.$http.put('/services/{service.id}', {
      data,
    });
```

Reply:
`$http` helper not used in this file, I'll rather match the code style of the file